src/Arcus.BackgroundJobs.AzureActiveDirectory/ClientSecretExpirationInfoProvider.cs

```diff
@@ -38,32 +38,26 @@ public ClientSecretExpirationInfoProvider(GraphServiceClient graphServiceClient,
         /// <param name="expirationThreshold">The threshold for the expiration, if the end datetime for a secret is lower than this value a <see cref="CloudEvent"/> will be published.</param>
         /// <returns>A list of applications that have expired secrets or secrets that are about to expire.</returns>
         /// <exception cref="ArgumentOutOfRangeException">Thrown when the <paramref name="expirationThreshold"/> is less than zero.</exception>
-        public async Task<List<ApplicationWithExpiredAndAboutToExpireSecrets>> GetApplicationsWithExpiredAndAboutToExpireSecrets(int expirationThreshold)
+        public async Task<IEnumerable<AzureApplication>> GetApplicationsWithPotentialExpiredSecrets(int expirationThreshold)
```

Comment:
Changed this method to a more logical name, let me know if you don't agree.

Reply:
No issue, seems like a better name 👍🏻